exercises/js/slideshow.js

```diff
@@ -0,0 +1,73 @@
+function SlideShow(slideShowOptions) {
+  this.imagesContainer = slideShowOptions.imagesContainer;
+};
+
+SlideShow.prototype.init = function() {
+  this.images = this.imagesContainer.find('img');
+  this.slidesCount = this.images.length;
+  this.createSlider();
+  this.startSlideShow();
+};
+
+SlideShow.prototype.createSlider = function() {
+  this.slider = $('<div id=slider>');
+  this.appendClonedImages(this.images);
+  this.slides = this.slider.find('img');
```

Comment:
@shubhamjuneja11 This is redundant here and can be accessed elsewhere. Please check.